code/components/citizen-server-impl/src/state/ServerGameState.cpp

```diff
@@ -683,6 +685,13 @@ void ServerGameState::Tick(fx::ServerInstanceBase* instance)
 
 		auto slotId = client->GetSlotId();
 
+		float scopeRadius = serverScopeRadius;
+
+		if (auto clientScopeRadius = client->GetData("scopeRadius"); clientScopeRadius.has_value())
```

Comment:
Please don't use the data bag for something that might get looked up a lot! Add a field to ClientSyncData (with an external accessor in ServerGameState.cpp?), perhaps.

Reply:
Alright